TuPedido/Services/Order/OrderService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using TuPedido.Helpers;
+using TuPedido.Models;
+
+namespace TuPedido.Services
+{
+    public class OrderService : IOrderService
+    {
+        private readonly IDropboxServiceClient dropboxService;
+        private readonly IExcelHelper excelHelper;
+        private readonly IConfiguration configuration;
+
+        public OrderService(IDropboxServiceClient dropboxService, IExcelHelper excelHelper, IConfiguration configuration)
```

Comment:
Tal vez lo hiciste por algo en especial, pero you dejaria un constructor sin parametros y cargaria esas interfaces con injections